R/conttables.b.R

```diff
@@ -643,7 +647,8 @@ contTablesClass <- R6::R6Class(
             if (oneResidualSelected) {
                 selectedIndex   <- which(residualSelections)
                 singleResTitle  <- subTitlesPh[selectedIndex]
-                postHoc$setTitle(jmvcore::format("Post Hoc Test ({title} Residuals)", title=singleResTitle))
+                phTitle <- jmvcore::format('Post Hoc Test (Type: {title})', title=singleResTitle)
+                postHoc$setTitle(.(phTitle))
```

Comment:
The translation should be done on the original pre-formatted string (as the translation is done without running the code). Also, I don't think the `Type` is necessary here. So it would be something like this:
```suggestion
                phTitle <- jmvcore::format(.('Post Hoc Test ({title})'), title=singleResTitle)
                postHoc$setTitle(phTitle)
```